src/sdk.ts

```
@@ -162,10 +281,103 @@ export class BiometrySDK {
+      headers['X-Device-Info'] = JSON.stringify(props.deviceInfo);
+    }
+
+    const response = await this.request<{ data: DocAuthInfo, message: string }>(
+      '/api-gateway/check-doc-auth',
+      'POST',
+      formData,
+      headers
+    );
+
+    return response.data;
+  }
```

Biome (1.9.4)

[error] Duplicate class member name "checkDocAuth"



(lint/suspicious/noDuplicateClassMembers)